corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -276,6 +277,28 @@ def test_error_publishing_to_kafka(self):
         self.assertFalse(form.is_error)
         self.assertTrue(form.initial_processing_complete)
 
+    def test_error_writing_to_blobdb(self):
+        error = ConnectionClosedError(endpoint_url='url')
+        with self.assertRaises(ConnectionClosedError), patch.object(get_blob_db(), 'put', side_effect=error):
+            self._submit('form_with_case.xml')
+
+        stubs = UnfinishedSubmissionStub.objects.filter(domain=self.domain, saved=False).all()
+        self.assertEqual(1, len(stubs))
```

Comment:
nit: This is vulnerable to failure if another test does not clean up after itself, leaving other stubs in the db. Does this stub have an id (possibly _ad38211be256653bceac8e2156475666_)? If it does, then one way to work around the vulnerability would be to check if a stub exists having that expected id.

Reply:
done

Reply:
updated